### PR TITLE
Class component in React

### PR DESCRIPTION
**Pull request by Ehraz98**

### CODE REVIEW

File: Day4/ProfileComponent.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        return (
+            <div>
+                <img src={avatar_url}/>
+                <p>{login}</p>
+                <p>{company}</p>
+            </div>
+        );
+    }
+
+    async componentDidMount(){
```
Comment (ketanmalik) [inline Day4/ProfileComponent.js]: You can keep all the functions before the render function - just for better readability.

**Review by ketanmalik — APPROVED** [review]
Looks good

File: Day4/App.js [hunk 2]
```diff
@@ -4,9 +4,16 @@ import { title } from './constants.js';
 import SearchBar from './SearchBar.js';
 import { useEffect, useState } from 'react';
 import NoResultsComponent from './NoResultsComponent.js';
-import { createBrowserRouter, RouterProvider, Outlet, Link } from 'react-router-dom';
+import {
```
Comment (SumitARG) [inline Day4/App.js:3]: Just for readability, try to push all your global import to the top. Kind of in a sequence like -> global imports, module imports, css imports and so on

Comment (Ehraz98) [reply Day4/App.js:3]: Resolved

File: Day4/ProfileComponent.js (hunk 1 above)
Comment (Ehraz98) [reply Day4/ProfileComponent.js]: Resolved

**Pull request merged**